GitUI/BranchTreePanel/StashTree.cs

```diff
@@ -63,6 +62,28 @@ private Nodes FillStashTree(IReadOnlyList<GitRevision> stashes, CancellationToke
             return nodes;
         }
 
+        internal void UpdateVisibility()
+        {
+            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
```

Comment:
❔ `Invoke`

Reply:
Make this an EventHandler?
What is the benefit in this scenario?

Reply:
I think `Invoke` is the short of `RunAsync(await SwitchToMainThread(); actual call;)`. But I don't mind using JTF for all async operations.